src/Validation/MultipleValidationWithAnd.php

```diff
@@ -25,7 +25,7 @@ class MultipleValidationWithAnd implements EmailValidation
     /**
      * @var EmailValidation[]
      */
-    private $validations = [];
+    private $validations;
```

Comment:
Please keep it as an empty array.

Reply:
The initializer is redundant as the constructor immediately overwrites it.
We can either drop the redundant initializer or to make things more explicit, we could make this `private array $validations` and drop the comment above which will also be redundant then.

Reply:
Type, thanks.